roles/log_generator/templates/log_generator.yml

```diff
@@ -73,7 +73,11 @@ spec:
         args:
           - > 
 {% if workload_args.pod_count | default(1) | int > 1 %}
-            echo "Waiting for all pods to be Ready";
+{% for pods in workload_args.pod_count %}
```

Comment:
as mentioned in slack the pods are spun up in the role so you would have to loop the pods there..